pkg/router/functionHandler.go

```diff
@@ -42,6 +37,11 @@ import (
 	"github.com/fission/fission/pkg/redis"
 	"github.com/fission/fission/pkg/throttler"
 	"github.com/fission/fission/pkg/types"
+	"github.com/pkg/errors"
```

Comment:
Please sort the imports

Reply:
Yup will do that.

Reply:
Normally, the order of import follows the rule
```
built-in pkg

3rd pkg

fission pkg
```

Reply:
Yes, but If I do this `./hack/verify-gofmt.sh` will fail because it requires the imports to be sorted in alphabatical order. I have tried this already.

Reply:
Could you try with this? I can pass the `verify-gofmt.sh`
```
import (
	"bytes"
	"context"
	"fmt"
	"io"
	"io/ioutil"
	"math/rand"
	"net"
	"net/http"
	"net/http/httputil"
	"net/url"
	"time"

	"github.com/pkg/errors"
	"go.opencensus.io/plugin/ochttp"
	"go.uber.org/zap"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
	k8stypes "k8s.io/apimachinery/pkg/types"

	fv1 "github.com/fission/fission/pkg/apis/fission.io/v1"
	"github.com/fission/fission/pkg/crd"
	ferror "github.com/fission/fission/pkg/error"
	"github.com/fission/fission/pkg/error/network"
	executorClient "github.com/fission/fission/pkg/executor/client"
	"github.com/fission/fission/pkg/redis"
	"github.com/fission/fission/pkg/throttler"
	"github.com/fission/fission/pkg/types"
)
```

Reply:
Yes this is working.